osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -122,6 +122,17 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
                 sliderBonus = osuLastObj.TravelDistance / osuLastObj.TravelTime;
             }
 
+            // The spacing bonus in speed evaluation
+            double currFlowBonus = Math.Pow(osuLastObj.MinimumJumpDistance / 125, 3.6);
+            double prevFlowBonus = Math.Pow(osuLastLastObj.MinimumJumpDistance / 125, 3.6);
+            double flowBonus = Math.Max(prevFlowBonus, currFlowBonus);
+
+            // Part of the aiming difficulty for this object is accounted for in the speed evaluator, so reduce aim difficulty here
+            if (flowBonus < 1)
```

Comment:
This is intentionally _not_ reducing difficulty for the case where it receives the full bonus. Is this an assumption of "not all of the aim difficulty is accounted for if the distance reaches `single_spacing_threshold`"?